src/stateProviders/provideLoggedInUser.js

```diff
@@ -127,24 +118,7 @@ export default provideState({
             .filter((field) => field && field.name !== 'sets')
             .map((field) => field.name),
         )
-        .then((fields) => (state) => {
-          const userRole = user.roles ? user.roles[0] : null;
-          const userRoleProfileFields =
-            userRole && userRole !== 'research'
-              ? without(fields, 'institution', 'jobTitle')
-              : fields;
-          const profile = pick(omit(user, 'sets'), userRoleProfileFields);
-          const filledFields = Object.values(profile || {}).filter(
-            (v) => (isArray(v) && v.length) || (!isArray(v) && v),
-          );
-          const percentageFilled = filledFields.length / userRoleProfileFields.length;
-          if (state.loggedInUser && state.percentageFilled < 1 && percentageFilled >= 1) {
-            trackUserInteraction({
-              category: TRACKING_EVENTS.categories.user.profile,
```

Comment:
its ok to remove tracking?

Reply:
In this case, yes. The concept of 'at what % the profile is completed' doesn t exist anymore.